src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandlerImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandlerImpl implements OperationHandler {
+    private FruitDao fruitDao = new FruitDaoImpl();

```

Comment:
it's better to use constructor injection

Reply:
I don't know what is this because it's the next topic